products/centos-stream.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CentOS Stream
+category: os
+iconSlug: centos-stream
+permalink: /centos-stream
+versionCommand: lsb_release --release
+releasePolicyLink: https://wiki.centos.org/About/Product
+releaseLabel: "CentOS Stream __RELEASE_CYCLE__"
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   cpe: cpe:/o:centos:centos
+-   cpe: cpe:2.3:o:centos:centos
+
+releases:
+-   releaseCycle: "9"
+    releaseDate: 2021-09-15
+    support: 2027-05-31
+    eol: 2027-05-31
+    latest: "9"
+    link: https://blog.centos.org/2021/12/introducing-centos-stream-9/
+
+-   releaseCycle: "8"
+    releaseLabel: "CentOS Stream 8"
+    releaseDate: 2019-09-24
+    support: 2024-05-31
+    eol: 2024-05-31
+    latest: "8"
+    link: https://wiki.centos.org/Manuals/ReleaseNotes/CentOSStream
+
+---
+
+> [CentOS](https://centos.org/) is a Linux distribution that provides a free, enterprise-class,
+> community-supported computing platform functionally compatible with Red Hat Enterprise Linux.
+
+CentOS Stream is the upstream variant, and contains content that is planned for the next minor
+release of Red Hat Enterprise Linux.
+
+CentOS Stream only has major versions, no minor versions. Its support ends when its corresponding
+RHEL release leaves full support.
```

Comment:
```suggestion
RHEL release leaves full support.

For information about CentOS, go to [our CentOS page](/centos).
```

We could add a sentence like this one to allow users to easily navigate between those pages.